Library/Homebrew/rubocops/shared/string_content.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module StringContent
```

Comment:
Since `regex_match_group.rb` was integrated into `helper_functions.rb`, shouldn't this module be there as well?

Reply:
Yeah, might as well move this into `HelperFunctions`.

Reply:
Done!